docusaurus/docs/how-to-guides/panel-plugins/plugin-internationalization.md

```diff
@@ -0,0 +1,264 @@
+
+const getStyles = () => {
+  return {
+    wrapper: css`
+      font-family: Open Sans;
+      position: relative;
+    `,
+    svg: css`
+      position: absolute;
+      top: 0;
+      left: 0;
+    `,
+    textBox: css`
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      padding: 10px;
+    `,
+  };
+};
+
+export const SimplePanel: React.FC<Props> = ({ options, data, width, height, fieldConfig, id }) => {
+  const theme = useTheme2();
+  const styles = useStyles2(getStyles);
+
+  if (data.series.length === 0) {
+    return <PanelDataErrorView fieldConfig={fieldConfig} panelId={id} data={data} needsStringField />;
+  }
+
+  return (
+    <div
+      className={cx(
+        styles.wrapper,
+        css`
+          width: ${width}px;
+          height: ${height}px;
+        `
+      )}
+    >
+      <svg
+        className={styles.svg}
+        width={width}
+        height={height}
+        xmlns="http://www.w3.org/2000/svg"
+        xmlnsXlink="http://www.w3.org/1999/xlink"
+        viewBox={`-${width / 2} -${height / 2} ${width} ${height}`}
+      >
+        <g>
+          <circle data-testid="simple-panel-circle" style={{ fill: theme.colors.primary.main }} r={100} />
+        </g>
+      </svg>
+
+      <div className={styles.textBox}>
+        {options.showSeriesCount && (
+          <div data-testid="simple-panel-series-counter">
+            <Trans i18nKey="components.simpePanel.options.showSeriesCount">
+              Number of series: {{ numberOfSeries: data.series.length }}
+            </Trans>
+          </div>
+        )}
+        <Trans i18nKey="components.simpePanel.options.textOptionValue">
+          Text option value: {{ optionValue: options.text }}
+        </Trans>
+      </div>
+    </div>
+  );
+};
+```
+
+## Extract translated text automatically
+
+Use the `i18next` [parser](https://github.com/i18next/i18next-parser#readme) to sweep all input files, extract tagged `i18n` keys, and save the translations. 
+
+To do so, install the parser:
+
+```shell npm2yarn
+npm install i18next-parser
+```
+
+Next, create a configuration file (for example, `i18next-parser.config.js`) and configure it so the parser sweeps your plugin and extracts the translations into the `locales/[$LOCALE]/[your-plugin].json`:
+
+```js title="i18next-parser.config.js"
+const pluginJson = require('../plugin.json');
+
+module.exports = {
+  locales: ['en-US'], // Only en-US  is updated - Crowdin will PR with other languages
+  sort: true,
+  createOldCatalogs: false,
+  failOnWarnings: true,
+  verbose: false,
+  resetDefaultValueLocale: 'en-US', // Updates extracted values when they change in code
+  defaultNamespace: pluginJson.id,
+  input: ['../**/*.{tsx,ts}'],
+  output: 'src/locales/$LOCALE/$NAMESPACE.json',
+};
+```
+
+Finally, add `i18n-extract` to `package.json`:
+
+```json title="package.json"
+  "scripts": {
+    "i18n-extract": "i18next --config src/locales/i18next-parser.config.js",
+  },    
+```
+
```

Comment:
Maybe add an example on how to run the command and a describe how running that command will create the translation file.

```shell npm2yarn
npm run i18n-extract
```

Reply:
and maybe add an example of how that file would look like after the run? Let me know if this is to much information for the reader

```json
{
  "components": {
    "simpePanel": {
      "options": {
        "showSeriesCount": "Number of series: {{numberOfSeries}}",
        "textOptionValue": "Text option value: {{optionValue}}"
      }
    }
  }
}
```